conf/locale/ar/LC_MESSAGES/django.po

```diff
@@ -344,7 +344,7 @@ msgstr "المساق "
 #: common/djangoapps/course_modes/models.py
 #: common/lib/xmodule/xmodule/annotatable_module.py
 #: common/lib/xmodule/xmodule/capa_base.py
-#: common/lib/xmodule/xmodule/discussion_module.py
+#: common/lib/xblock/xblock_discussion/xblock_discussion.py
```

Comment:
Don't check in changes to the locale files. There is a separate process for updating these in Transifex.